src/reviewList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react"
+function ReviewList(review_id){
+const [reviews, setReviews] = useState([])
+    fetch('https://jamies-backend-project.onrender.com/api/reviews')
```

Comment:
You will be wanting to use a useeffect and and api file for fetching data - currently I can't load anything as it seems you have an infinite loop.